positron/modules/process.js

```diff
@@ -6,8 +6,24 @@
 
 const { classes: Cc, interfaces: Ci, results: Cr, utils: Cu } = Components;
 
+// The Node version of this module only re-exports the `process` global,
+// because the global is implemented in natively.  But we implement the global
```

Comment:
> Typo? "implemented in natively"?

Nice catch! I was wordsmithing that comment at the last minute, so I let that one slip through. Fixed in d16a2a0.